libraries/troubleshoot/impl/src/main/kotlin/io/element/android/libraries/troubleshoot/impl/history/PushHistoryPresenter.kt

```diff
@@ -28,13 +29,21 @@ class PushHistoryPresenter @Inject constructor(
     override fun present(): PushHistoryState {
         val coroutineScope = rememberCoroutineScope()
         val pushCounter by pushService.pushCounter.collectAsState(0)
-        val pushHistory by remember {
+        var showOnlyErrors: Boolean by remember { mutableStateOf(false) }
+        val pushHistory by remember(showOnlyErrors) {
             pushService.getPushHistoryItemsFlow()
+        }.map {
+            it.filter { item ->
+                showOnlyErrors.not() || item.hasBeenResolved.not()
+            }
```

Comment:
Shouldn't this be inside the `remember` block?

Reply:
I see this was fixed in the latest commit.

Reply:
Yeah, good catch, lint caught that mistake as well